docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -3,4 +3,140 @@ title: Test Replay
 sidebar_position: 20
 ---
 
-Test Replay!
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What Test Replay is and why you should use it
+- How to get started with Test Replay in Cypress Cloud
+- How to debug failed and flaky CI test runs more accurately and efficiently
+
+:::
+
+:::caution
+
+{/* todo: change changelog url to real page when it's ready/won't break build */} <br/>
+&#8239;<Icon name="exclamation-triangle" /> Test Replay is available when recording tests to Cypress Cloud using Cypress `v13` and up. See the [Cypress v13 changelog](/guides/references/changelog#12-0-0).
+
+:::
+
+:::caution
+
+Test Replay currently supports Chromium-based browsers only.
+
+:::
+
+## What Is Test Replay?
+
+Test Replay in Cypress Cloud allows developers to accurately and efficiently debug failed and flaky CI test runs. It captures every detail of test runs, enabling developers to replay and interact with the DOM, inspect network events, and console events as they happened during CI. This eliminates the need to replicate failed tests locally and enhances the ability to debug complex applications with confidence and speed.
+
+### Test Replay vs Screenshots and Videos
+
+Before Cypress `v13`, test failures in CI have historically been captured through screenshots, videos, and stack trace outputs, but these artifacts provide limited information. Developers often need more context to understand the root cause of failures, requiring them to reproduce the issue locally, which may not always be possible due to environmental or network conditions. This knowledge gap between test failures and debugging capabilities leads to increased debugging time and potentially deploying unresolved bugs to production.
+
+### How Does Test Replay Advance Your Workflow?
+
+Test Replay complements the existing visual artifacts provided by Cypress Cloud by allowing developers to directly interact with tests as they occurred in CI. It eliminates the need for local test replication by enabling developers to time-travel to the exact moment of test failure, replaying and interacting with network requests, console logs, JavaScript errors, and element rendering. With Test Replay, developers gain access to previously inaccessible details and feedback loops, enabling faster and more accurate debugging, and instilling confidence in handling test failures in CI. Replacing video recordings with Test Replay provides a superior debugging experience.
+
+## Getting Started
+
+:::tip
+
+&#8239<Icon name="check-circle" /> Test Replay is available to **all** Cypress Cloud plans at no additional cost, subject to usage limits. Free plans allow up to 500 test results per month, while exceeding this limit requires upgrading the plan.
```

Comment:
Maybe link out to pricing or something here – not sure what pattern we use elsewhere

Reply:
No real consistent pattern for this. I'll add the basic line from the FAQs